lua/lint/linters/glslc.lua

```diff
@@ -0,0 +1,30 @@
+-- path/to/file:line: severity: 'offending part of source code': message
+local pattern = "%s*([^:]+):(%d+): (.+): (.+: .+)"
+local groups = { "file", "lnum", "severity", "message" }
+
+local tmp_file
+if vim.fn.fn.has "win32" == 1 then
+  tmp_file = "/dev/null"
+else
+  local cache_dir = vim.fn.stdpath("cache")
+  if vim.loop.fs_stat(cache_dir) then
+    tmp_file = cache_dir .. "\\nvim-lint_glslc.spv"
+  else
+    error('Could not find folder of temporary file for glslc linter')
+  end
+end
+
+return {
+  cmd = "glslc",
+  stdin = false,
+  args = {"-o", tmp_file},
```

Comment:
`tmp_file` could maybe be provided by the plugin if needed by more linters.

Reply:
How difficult would it be to get a change into this linter to not require the output file? 

This is the first linter I encounter that requires something like this. 

(Not a blocker for this PR, but I wouldn't extend the framework to support this more easily)